rpm/felix.spec.in

```diff
@@ -116,7 +116,7 @@ install etc/*.cfg.example %{buildroot}%{_sysconfdir}/calico
 install -d %{buildroot}%{_datadir}/calico/bird
 install etc/bird/*.template %{buildroot}%{_datadir}/calico/bird
 install -d %{buildroot}%{_bindir}
-install -m 755 etc/*.sh %{buildroot}%{_bindir}
+install -m 755 etc/calico-diags etc/*.sh %{buildroot}%{_bindir}
```

Comment:
Should this file be moved to the packaging repo? I kind of feel like it should.

Reply:
It certainly could be, but the spec file is repo-specific, and I think it goes with the repo because adding dependencies and new installed files here is quite similar to adding those into our container builds.  If a Felix enhancement, for example, requires some new dependency, I think it makes sense to be able to make all of those changes together in the same PR.